conformance_tests/sysman/test_sysman_frequency/CMakeLists.txt

```diff
@@ -26,3 +26,5 @@ add_lzt_test(
     sysman_matrix_multiplication
   DEFINES USE_ZESINIT  
 )
+
+add_definitions(-DMATRIX_MUL_SPV_PATH="${CMAKE_SOURCE_DIR}/conformance_tests/sysman/test_sysman_frequency/kernels/sysman_matrix_multiplication.spv")
```

Comment:
This cannot work properly - `MATRIX_MUL_SPV_PATH` will be invalid after installation. Please see `add_lzt_test` cmake function and how `KERNELS` specifier works.